src/pages/SonarFeed/SonarFeedPage.module.scss

```diff
@@ -1,35 +1,48 @@
+@import 'variables.scss';
+@import "~@santiment-network/ui/mixins.scss";
+
 .header {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    margin-bottom: 22px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 22px;
 }
 
 .tabs {
-    display: block;
-    margin-bottom: 40px;
+  display: block;
+  margin-bottom: 40px;
 }
 
 .newSignal {
-    background: var(--green) !important; // NOTE(vanguard): Delete after updating ui button green variant
-    color: var(--white) !important;
-    fill: var(--white) !important;
-
-    &__icon {
-        height: 18px;
-        margin-right: 7px;
-    }
+  background: var(--green) !important; // NOTE(vanguard): Delete after updating ui button green variant
+  color: var(--white) !important;
+  fill: var(--white) !important;
```

Comment:
This css is no longer necessary. `.newSignal` button could be defined with following props: `fill`, `accent='positive'`